changes/4511.fixed

```diff
@@ -0,0 +1 @@
+Provides ability to overload the default `PERMISSIONS_ACTION_MAP` for a viewset.
```

Comment:
Needs rewording now I think?

Reply:
Will do pending outcome of how to implement. :-)